startup/52-hdcmscans.py

```diff
@@ -335,12 +341,18 @@ def peakup_fine(scaler='sclr_i0', plot=True, shutter=True, use_calib=True,
             yield from bps.mov(dcm.c2_pitch_kill, 1.0)
 
     # Set counting time
-    sclr1.preset_time.put(1.0)
-
+    sclr1.preset_time.put(dwell)
+
+    # Set metadata
+    scan_md = get_stock_md({})
+    scan_md['scan']['type'] = 'PEAKUP'
+    scan_md['scan']['detectors'] = det_names
+    scan_md['scan']['dwell'] = dwell
+    
     # Open the shutter
     # if (shutter == True):
     #     yield from bps.mov(shut_b, 'Open')
-    yield from check_shutters(shut_b, 'Open')
+    yield from check_shutters(shutter, 'Open')
```

Comment:
Yes, it was a mistake to have the shut_b object there. Basically, we set a shutter bool to control the shutters. To make that coding easier, we send that bool to the check_shutters function and if it is true then it will open/close them, if it is false, it will pass through.